asyncpg/connect_utils.py

```diff
@@ -442,6 +458,21 @@ def _parse_connect_dsn_and_args(*, dsn, host, port, user,
     else:
         sslmode = SSLMode.disable
 
+    if sslcert is None:
+        sslcert = os.getenv('PGSSLCERT')
+
+    if sslkey is None:
+        sslkey = os.getenv('PGSSLKEY')
+
+    if sslrootcert is None:
+        sslrootcert = os.getenv('PGSSLROOTCERT')
+
+    if isinstance(ssl, ssl_module.SSLContext):
+        if sslcert:
+            ssl.load_cert_chain(sslcert, keyfile=sslkey)
+        if sslrootcert:
+            ssl.load_verify_locations(cafile=sslrootcert)
```

Comment:
`load_verify_locations` also supports CRL (just call it again), so let's add that while we are here.